app/assets/src/components/views/bulk_download/BulkDownloadModal.jsx

```diff
@@ -193,6 +195,12 @@ class BulkDownloadModal extends React.Component {
 
   handleFieldSelect = (downloadType, fieldType, value, displayName) => {
     this.setState(prevState => {
+      logAnalyticsEvent("BulkDownloadModal_field-for-download-type-selected", {
```

Comment:
you should have it as `_selected` instead of `-` for the verb for each of these (https://czi.quip.com/67RCAIiHN0Qc/IDseq-product-analytics-How-to-log)

Reply:
Thanks for point this out!